src/consensus/aft/raft_types.h

```diff
@@ -88,6 +114,8 @@ namespace raft
     raft_append_entries_response,
     raft_request_vote,
     raft_request_vote_response,
+
+    bft_Request,
```

Comment:
Why is `bft_request` under `RaftMsgType`?

Reply:
the word raft is not correct. At this point of type our CFT consensus is not raft. This message type is only used for bft, and while i am removing the name raft as I go I do not want to change the entire codebase all at once.